packages/commonwealth/server/routes/migrateEvent.ts

```diff
@@ -0,0 +1,63 @@
+import { success, TypedRequestBody, TypedResponse } from '../types';
+import { AppError } from '../util/errors';
+import { DB } from '../database';
+import {
+  migrateChainEntities,
+  migrateChainEntity,
+} from '../scripts/migrateChainEntities';
+
+enum MigrateEventErrors {
+  Failed = 'Request Failed.',
+  AllError = 'Failed to migrate all events.',
+  MissingChainID = 'Missing chain_id.',
+  ChainError = 'Failed to migrate chain events.',
+}
+
+type migrateEventReq = {
+  migrateAll: boolean;
```

Comment:
Can you make `migrateAll` false by default (optional)? Then we should ensure it's either passed explicitly as true, or that a chain_id is provided.